server/src/test/java/com/google/coffeehouse/common/PersonTest.java

```diff
@@ -39,20 +38,15 @@
   private static final String ALT_EMAIL = "New Email";
   private static final String PRONOUNS = "she/her";
   private static final String ALT_PRONOUNS = "New Pronouns";
-  private static final String IDENTIFICATION_STRING = "predetermined-identification-string";
+  private static final String USERID = "predetermined-identification-string";
```

Comment:
Nit: As UserId is treated as two separate words below, consider changing this to USER_ID.

Reply:
Absolutely correct, I've switched all instances of `USERID` in this PR to `USER_ID`.